daprdocs/content/en/operations/performance-and-scalability/perf-service-invocation.md

```diff
@@ -59,10 +59,10 @@ When running in a highly available production setup, the Dapr control plane cons
 
 | Component  | vCPU | Memory
 | ------------- | ------------- | -------------
-| Operator  | 0.001  | 12.5 Mb
-| Sentry  | 0.005  | 13.6 Mb
-| Sidecar Injector  | 0.002  | 14.6 Mb
-| Placement | 0.001  | 20.9 Mb
+| Operator  | 0.003  | 18 Mb
```

Comment:
Is this the only place that we capture the Control Plane numbers and are these consistently the same for all the performance test? If so we should separate these out. If not, then should each perf test have different control plane numbers like this one?